.github/workflows/publish.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     - name: Build and Push Gem
       env:
         GEM_HOST_API_KEY: ${{ secrets.RUBYGEM_API_KEY }}
-      run: >-
+      run: |
         gem build vellum_ai.gemspec
+        
         gem push vellum_ai-*.gem --host https://rubygems.org
```

Comment:
Could we add a workflow_dispatch so that we could run it this one time? otherwise the options are:
- cut a new release
- run it locally

Reply:
Yup, let me add and we can remove it later